sql/catalyst/src/main/java/org/apache/spark/sql/connector/read/Statistics.java

```diff
@@ -31,4 +35,7 @@
 public interface Statistics {
   OptionalLong sizeInBytes();
   OptionalLong numRows();
+  default Optional<Map<NamedReference, ColumnStatistics>> columnStats() {
```

Comment:
shall we use empty map to indicate no column stats? Catalyst column stats also use map directly.